package Task;

public class TaskNotFoundException extends Exception {    //Класс-исключение "Задача не найдена" наследуемая от проверяемого исключения
}
